rpcs3/Input/ds4_pad_handler.cpp

```diff
@@ -728,6 +751,7 @@ ds4_pad_handler::DS4DataStatus ds4_pad_handler::GetRawData(const std::shared_ptr
 		return DS4DataStatus::NoNewData;
 
 	int battery_offset = offset + DS4_INPUT_REPORT_BATTERY_OFFSET;
+	device->is_initialized = true;
```

Comment:
you never set this to false (for example before the second ReadError return above)

Reply:
It's declared as false.

Reply:
devices can disconnect and reconnect

Reply:
You're right, thanks!